styles.css

```diff
@@ -2,21 +2,24 @@
 	--color-primary: #0e345a;
 	--color-primary-300: #26486b;
 	--color-primary-200: #879aad;
+	--color-bg-scrollbar: #aaaaaa;
+	--color-border-scrollbar: rgba(0, 0, 0, 0);
 }
 
 ::-webkit-scrollbar {
 	width: 10px;
 }
 
 ::-webkit-scrollbar-thumb {
-	border: 1px solid rgba(0, 0, 0, 0);
+	border: 1px solid var(--color-border-scrollbar);
 	background-clip: padding-box;
 	border-radius: 9999px;
-	background-color: #aaaaaa;
```

Comment:
use color variables. also, why have you given such a high border radius?

Reply:
Oh, about this, well I was experimenting with the scrollbar last night and completely forgot about it. Will change this ASAP